data/kubermatic/master/runbook.json

```diff
@@ -887,7 +887,7 @@
         {
           "alert": "PromBadConfig",
           "annotations": {
-            "mesage": "Prometheus failed to reload config.",
+            "message": "Prometheus failed to reload config.",
```

Comment:
NB: This must be fixed upstream (Kubermatic repo), or else the next sync will overwrite this fix.

Reply:
I'm going to add the same job in kubermatic repo anyway